### ISSUE TITLE
Libdoc: Allow switching between compact and expanded list of shortcuts and tags

### ISSUE THREAD

**Issue by Rudolf-AT** [robotframework/robotframework#3635]
Hi,

an html view of a library documentation contains a very compact list of all keywords of that library in the "Shortcuts" section.

I would like to have an option on the html page to list the keywords one per line, because for me it seems easier to read.
Example:
![RF_shortcuts-list-style-compact_-1](https://user-images.githubusercontent.com/69890168/90791387-6b24b500-e309-11ea-94ee-ce602cb3b3e4.png)

![RF_shortcuts-list-style-expanded_-1](https://user-images.githubusercontent.com/69890168/90791279-4cbeb980-e309-11ea-8564-228688dabcef.png)

There is already a pull request [https://github.com/robotframework/robotframework/pull/3634](https://github.com/robotframework/robotframework/pull/3634) which implements this example.

Best Regards,
Rudolf

PS: Thanks for all your work!

**Comment by pekkaklarck** [comment]
This was implemented by PR #3634 thanks to @Rudolf-AT. I did some fine tuning to styles and add same functionality to tags in ab04045095f5cea2ddaeee1a61eea27ac0ca08cd and made the list style selection persistent in f06de6c46164e92c20b1b27b0eeb0ab6cd807257.

I hope @Rudolf-AT can review the changes and @Snooz82 might be interest to test this as well. I close this issue but we can open it if there are bugs or if the functionality could still be enhanced.

Thanks again for the PR and the great idea @Rudolf-AT!
